CHANGES.md

```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.365
+- Update model version v0.0.312
+  - Added support for new endpoints and api updates for `addons_mgmt`
```

Comment:
can you be specific as to what endpoints were added please :pray:

Reply:
specified all the changes that were made 🙂